src/game/SpellAuras.cpp

```diff
@@ -4396,20 +4396,17 @@ void Aura::PeriodicTick()
                 // Check caster has one of the ranks of Improved Mend Pet
                 if (pCaster->HasSpell(19572) || pCaster->HasSpell(19573))
                 {
-                    // The first tick is fired when channel starts, so only check other ticks
-                    if (GetAuraTicks() > 1)
-                    {
-                        int32 triggerAmount = 0;
+                    int32 triggerAmount = 0;
 
-                        // Set threshold for dispel check
-                        if (pCaster->HasSpell(19572))
-                            triggerAmount = 15; //rank 1
+                    // Set threshold for dispel check
+                    if (pCaster->HasSpell(19572))
+                        triggerAmount = 15; //rank 1
 
-                        if (pCaster->HasSpell(19573))
-                            triggerAmount = 50; //rank 2
+                    if (pCaster->HasSpell(19573))
+                        triggerAmount = 50; //rank 2
 
-                        if (roll_chance_i(triggerAmount))
-                            pCaster->CastSpell(target, 24406, true, nullptr, this);
+                    if (roll_chance_i(triggerAmount)) {
```

Comment:
What's { doing at the end of that line? Also, why is the brackets needed at all on that single liner?